src/subcommands/util.rs

```diff
@@ -370,14 +361,28 @@ message = "0x"
                     &message,
                 )?;
                 let result = serde_json::json!({
+                    "message": format!("{:#x}", message),
                     "signature": format!("0x{}", hex_string(&signature).unwrap()),
                     "recoverable": recoverable,
                 });
                 Ok(result.render(format, color))
             }
             ("verify-signature", Some(m)) => {
-                let message: H256 =
-                    FixedHashParser::<H256>::default().from_matches(m, "message")?;
+                let binary_opt = HexParser.from_matches_opt(m, "binary-hex", false)?;
+                let message_str_opt : Option<String> = NullParser.from_matches_opt(m, "message", false)?;
+                let magic_string = String::from("Nervos Message:");
+                let magic_bytes = magic_string.as_bytes();
+
+                //TODO: Handle case when BOTH exist
```

Comment:
Isn't this precluded/handled already by what we have above? Can't submit a PR against upstream with a TODO.